beetmoverscript/src/beetmoverscript/script.py

```diff
@@ -498,7 +498,7 @@ async def move_partner_beets(context, manifest):
             destination = get_destination_for_partner_repack_path(context, manifest, full_path_artifact, locale)
 
             # S3 upload
-            if context.config["clouds"]["aws"][context.resource]["enabled"]:
+            if context.config["clouds"]["aws"][context.resource]["enabled"] is True:
```

Comment:
Do we actually use this part? I don't see buildhub files in directories like https://archive.mozilla.org/pub/firefox/candidates/125.0-candidates/build1/partner-repacks/acer/acer-002/v1/win64/en-US/. (It's probably fine to have this change anyways...just wondering...)

Reply:
This change makes it so we don't upload partner repacks to aws.

I was actually tempted to just remove the s3 upload bits from move_partner_beets and push_to_releases entirely, but figured I'd start small.

Reply:
Oh duh, of course. Sorry!